app/controllers/concerns/sms.rb

```diff
@@ -2,36 +2,16 @@
 require "twilio-ruby"
 
 # A set of utility methods used in relation to SMS notifications
-# @author Eric Schlange <eric.schlange@northwestern.edu>
 module Sms
-  extend ActiveSupport::Concern
-
-  # Send an SMS message body to a specified receiving participant (leaving
-  # off recipient defaults the value to the current participant)
-  # rubocop:disable Metrics/AbcSize
-  def send_sms(recipient = current_user, message_body)
+  def send_sms(recipient, message_body)
+    return unless Rails.env.staging? || Rails.env.production?
```

Comment:
I think combining the exit logic is fine...except in cases where the line gets too complex or unwieldy.

Also, I'm confused by the link you provided. The blog post author seems to be arguing FOR multiple returns and AGAINST a single guard clause. He suggests that it is more readable and testable to have multiple returns.